src/chunk_index.c

```diff
@@ -107,6 +107,7 @@ adjust_expr_attnos(Oid ht_relid, IndexInfo *ii, Relation chunkrel)
 
 		char *attname = get_attname(ht_relid, var->varattno, false);
 		var->varattno = get_attnum(chunkrel->rd_id, attname);
+		var->varattnosyn = get_attnum(chunkrel->rd_id, attname);
```

Comment:
```suggestion
#if PG13_GE
		var->varattnosyn = var->varattno;
#else
		var->varoattno = var->varattno;
#endif
```
varattnosyn was named varoattno before PG13